idd/Energy+.idd.in

```diff
@@ -49467,13 +49467,106 @@ Coil:Cooling:Water:DetailedGeometry,
   A7 , \field Condensate Collection Water Storage Tank Name
        \type object-list
        \object-list WaterStorageTankNames
-  N17; \field Design Water Temperature Difference
+  N17, \field Design Water Temperature Difference
        \type real
        \units deltaC
        \minimum> 0.0
        \note This input field is optional. If specified, it is used for sizing the Design Water Flow Rate.
        \note If blank or omitted, the Loop Design Temperature Difference value specified in Sizing:Plant
        \note object is used for sizing the Design Water Flow Rate.
+  N18; \field Design Water Inlet Temperature
+       \type real
+       \units C
+       \minimum> 0.0
+       \note This input field is optional. If specified, it is used for sizing the coil Design Geomtery
+       \note Parameters. If blank or omitted, the Design Loop Exit Temperature value specified in 
+       \note Sizing:Plant object is used for sizing the coil Design Geomtery Parameters.
```

Comment:
The new input field.

Reply:
This should probably  be \autosizable with a default of autosize, like the same field in Coil:Cooling:Water.

Reply:
I believe we are talking about this field being blank or autosizable. I'd rather see autosizable and if not an economizer coil just read from the plant sizing object. If it is an economizer coil then use a 10C detlaT (or whatever is reasonable for this application) from what will be used for sizing (e.g., final sys sizing pre-cool temp - 10 or something like that, or maybe a fraction of OAT - precool temp so coil size changes by geography). I see Design Inlet Water Temperature in the Coil:Cooling:Water object, probably should use the same field name.

Reply:
Renamed the field to *Design Inlet Water Temperature*.

Reply:
I am going to set the *Des Inlet Water Temperature* to *Design Supply Air Temperature* minus 5.0 if the water cooling coil is used as water-side economizer as shown below. The same logic will be used for *simple* and *detailed* water cooling coils if the coil is used as water side economizer. @rraustad @mjwitte your thought?

```
            if (state.dataWaterCoils->WaterCoil(CoilNum).UseAsWaterSideEconomizer) {
                if (state.dataWaterCoils->WaterCoil(CoilNum).DesInletWaterTemp == AutoSize) TempSize = (state.dataSize->DataDesOutletAirTemp - 5.0);
            }
```

Reply:
How would you determine `UseAsWaterSideEconomizer`? I'm not sure we need to determine that. Perhaps a better test would be
if DesInletWaterTemp > DataDesOutletAirTemp 
then throw a warning and set it to DataDesOutletAirTemp  - 5.0

Reply:
Currently, the CoilSystem:Cooling:Water object is used as water side economizer and set the water coil as water-side economizer.  If we are planning to use this new coilsystem object for other applications, then this may not work.

```
                // set that the cooling coil is economizer
                if (thisSys.m_CoolingCoilIndex > 0) {
                    state.dataWaterCoils->WaterCoil(thisSys.m_CoolingCoilIndex).UseAsWaterSideEconomizer = true;
                }
```

Reply:
UseAsWaterSideEconomizer approach limits the CoilSystem use for water-side economizer only. I will make changes per @mjwitte suggestion as follows.  

```
            if (state.dataWaterCoils->WaterCoil(CoilNum).DesInletWaterTemp > state.dataSize->DataDesOutletAirTemp) {
                state.dataWaterCoils->WaterCoil(CoilNum).DesInletWaterTemp = state.dataSize->DataDesOutletAirTemp - 5.0;
                ShowWarningError(state, "Invalid design inlet water temperature for Coil:Cooling:Water " + state.dataWaterCoils->WaterCoil(CoilNum).Name);
                ShowContinueError(state, "coil design inlet water temperature should be less than the coil design outlet air temperature");
                ShowContinueError(state, "coil design inlet water temperature is set to the coil design outlet air temperature - 5.0C");
            }
```

Reply:
Sample warning message:
```

   ** Warning ** Invalid design inlet water temperature for Coil:Cooling:Water = FREE COOLING COIL
   **   ~~~   ** ...design inlet water temperature = 11.000 C
   **   ~~~   ** ...design outlet air temperature = 16.000 C
   **   ~~~   ** ...design inlet water temperature should be less than the design outlet air temperature
   **   ~~~   ** ...design inlet water temperature is set to the design outlet air temperature minus 5.0C
```

Reply:
I was thinking that you know it is an economizer coil if it is connected to the condenser plant and/or if the Minimum Air To Water Temperature Offset > 0.

Reply:
> I was thinking that you know it is an economizer coil if it is connected to the condenser plant and/or if the Minimum Air To Water Temperature Offset > 0.

I'd rather not have any magic behind the scenes. We want this to be general use. I suppose the warning could mention that there is an input field to set this directly. And if the -5.0C is too arbitrary, maybe we just make this a severe/fatal and say use this field to fix it.

Reply:
I agree with @mjwitte, we should try to avoid adding a logic that is not obvious to the user. We have input fields that the user can use in the *simple* and *detailed* water cooling coil objects. May be we should add description in the input field how it is used when these coils are used as water-side economizer.

Reply:
Below is sample text description hat we can add to the idd for these coil types:

```
  N18; \field Design Inlet Water Temperature
       \type real
       \units C
       \autosizable
       \default autosize
       \minimum> 0
       \note This input field is optional. If specified, it is used for sizing the coil Design Geomtery
       \note Parameters. If blank or omitted, the Design Loop Exit Temperature value specified in 
       \note Sizing:Plant object is used for sizing the coil Design Geomtery Parameters. 

       \note If this coil is used as waterside economizer (connected to demand side of a condenser Loop), 
       \note the autosized value, which is the condenser loop Design Loop Exit Temperature value, can be
       \note larger than the coil design outlet air temperature. In this case the design inlet water 
       \note temperature value is reset to coil design outlet air temperature minus 5.0 DeltaC. User can 
       \note also specify appropriate design value in this field.
```

Reply:
Looking at this again, since we made this autosizable, you can replace "If blank or omitted" with "If autosized, ...".
I wouldn't qualify the second part with "if this coil is used as a waterside economizer .....". There could be other reason that the design inlet water temp is higher than the leaving air temp. So, you could just start with 
"If the autosized value is higher than the coil design outlet air temperature, then the design inlet water temperature value is reset . . . ."

Reply:
Done.